internal/transport/transport_test.go

```diff
@@ -667,7 +667,7 @@ func (s) TestClientMix(t *testing.T) {
 		<-ct.Error()
 		ct.Close(fmt.Errorf("closed manually by test"))
 	}(ct)
-	for i := 0; i < 1000; i++ {
+	for i := 0; i < 500; i++ {
```

Comment:
This runs for 5 seconds.  The server will stop after 5 seconds (see above).  Before it would be 10/5, seemingly doing this in order to test the RPCs after the server was stopped.  Maybe change this one to 2s and the above one to 1s?

Also, above can be replaced by `time.AfterFunc(whateverTimeInSeconds * time.Second, s.stop)`